.github/workflows/merged.yml

```
@@ -150,6 +150,12 @@ jobs:
           GIT_TOKEN: ${{secrets.MODULE_FETCH_TOKEN}}
         run: |
           ./scripts-configs/store-access-token.sh
+          
+      - name: Export AppCheck Secret
+        env:
+          APP_CHECK_DEBUG_TOKEN: ${{ secrets.BUILD_APPCHECK_DEBUG_TOKEN }}
+        run: |
+          echo APP_CHECK_DEBUG_TOKEN=${{ secrets.BUILD_APPCHECK_DEBUG_TOKEN }} >> ../secrets.xcconfig
```

So line 156, says:
- _take this secret `BUILD_APPCHECK_DEBUG_TOKEN` and put it in an environment variable named `APP_CHECK_DEBUG_TOKEN`_.

This is required because GitHub Secrets aren't available to the run scripts by default (see the docs I linked above).

Then line 158 says: 
1. _print out this string "APP_CHECK_DEBUG_TOKEN=$APP_CHECK_DEBUG_TOKEN"_ where the latter half is the environment variable we just created from the secret
2. then `>>` says instead of printing that out to the console, append it to a file named `secrets.xcconfig`.

```suggestion
        env:
          APP_CHECK_DEBUG_TOKEN: ${{ secrets.BUILD_APPCHECK_DEBUG_TOKEN }}
        run: |
          echo APP_CHECK_DEBUG_TOKEN=$APP_CHECK_DEBUG_TOKEN >> ../secrets.xcconfig
```

---

Ah okay, thank you!